packages/x-tree-view/src/internals/useTreeView/useTreeView.utils.ts

```diff
@@ -71,3 +76,14 @@ export const populateInstance = <T extends TreeViewAnyPluginSignature>(
 ) => {
   Object.assign(instance, methods);
 };
+
+export const populatePublicAPI = <T extends TreeViewAnyPluginSignature>(
+  publicAPI: TreeViewUsedPublicAPI<T>,
+  methods: T['publicAPI'],
+) => {
+  if (!publicAPI) {
```

Comment:
Is this condition useful? Since we have a initialization in `useTreeViewApiInitialization`